CHANGELOG.md

```diff
@@ -2,6 +2,12 @@
 
 ## Version 0.0.x
 
+### 0.0.14.x (2021-Jan-20)
+- Added LogDebugMessage callback to example
```

Comment:
Mention the new version of the CAS BACnet Stack that has been used with the LogDebugMessage

- Updated CAS BACnet Stack to version XXXXX